cli/src/torii/routing.rs

```diff
@@ -278,14 +276,16 @@ async fn handle_blocks_stream(kura: Arc<Kura>, mut stream: WebSocket) -> eyre::R
 
     let mut interval = tokio::time::interval(std::time::Duration::from_millis(10));
     loop {
+        // FIXME: cleanup.
```

Comment:
what's this about?

Reply:
Lots of refactoring that needs to be done in the loop.